tutorials/getstarted.qmd

```diff
@@ -1,3 +1,312 @@
 # 🚀 Get Started with LieGroups.jl
 
-In this tutorial, we aim to present how to get started with `LieGroups.jl`.
\ No newline at end of file
+## Introduction
+
+This tutorial introduces both a few basics of [Lie groups](https://en.wikipedia.org/wiki/Lie_group) as well as how to use [LieGroups.jl](https://juliamanifolds.github.io/LieGroups.jl/stable/). The Lie groups we consider are the rotations in the plane,
+or [special orthogonal group](../groups/special_orthogonal_group.md) as well as rigid body motions,
+or in other words the [special euclidean group](../groups/special_euclidean_group.md).
+
+In a nutshell a Lie group $𝒢$ is a [manifold](https://en.wikipedia.org/wiki/Manifold) $ℳ$ that also is a group a group operation.
+A manifold informally be described as a set that “locally looks like a Euclidean space”
+and has some way to measure angles at every point, to be precise an inner product, that is allowed to (smoothly)
+depend on the point we are at.
```

Comment:
This isn't necessary for Lie groups, so I'd suggest skipping the inner product here.

Reply:
After introducing the left invariant vector fields and identifying everything with the Lie algebra, I do agree – but I do that in the following